README.md

```diff
@@ -225,7 +225,23 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/fastruby/skunk/issues.
+
+
+Bug reports and pull requests are welcome on GitHub at .
+
+## Contributing
+
+Bug reports and pull requests are welcome on GitHub at [https://github.com/fastruby/skunk](https://github.com/fastruby/skunk). This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
```

Comment:
Maybe this should point to https://github.com/fastruby/skunk/blob/main/CODE_OF_CONDUCT.md?